common/models/poi-post.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "PoiPost",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "comment": {
```

Comment:
Any reason why we called this field `comment` instead of `post` again?

Reply:
Did we? I can change it now

Reply:
Oh no I see its `comment` in the ERD, I'm just saying the name might be a bit misleading. Points more to the comments on a post than to the post itself.

Reply:
we can go with message or info, I agree, `comment` might be confusing